src/_utils.js

```diff
@@ -591,7 +591,8 @@ export const processCss = (stylesInfo, options) => {
 
   if (useSourceMaps) {
     const generator = makeSourceMapGenerator(fileInfo.file)
-    const filename = fileInfo.sourceFileName
+    const filename = fileInfo.sourceFileName || fileInfo.filename
```

Comment:
The bug was here, these are not defined when not using the CLI I guess. Chances are that source maps still work when the plugin is used with babel loader or something.

Reply:
I started looking into this too and actually just opened a pr not seeing yours 🤦‍♂️

On my branch I used `file.opts.generatorOpts.sourceFileName` since it seems to match with the value we were expecting before. I closed it after seeing yours, but I just re-opened it so you can take a look.